bridges/zombienet/helpers/only-required-headers-synced-when-idle.js

```diff
@@ -0,0 +1,81 @@
+const utils = require("./utils");
+
+async function run(nodeName, networkInfo, args) {
+    const {wsUri, userDefinedTypes} = networkInfo.nodesByName[nodeName];
+    const api = await zombie.connect(wsUri, userDefinedTypes);
+
+    // parse arguments
+    const exitAfterSeconds = Number(args[0]);
+    const bridgedChain = require("./chains/" + args[1]);
+
+    // start listening to new blocks
+    let atLeastOneMessageReceived = false;
+    let atLeastOneMessageDelivered = false;
+    const unsubscribe = await api.rpc.chain.subscribeNewHeads(async function (header) {
+        const apiAtParent = await api.at(header.parentHash);
```

Comment:
For me it always fails here. Because it receives the genesis header, the parent hash is `0x0000000000000000000000000000000000000000000000000000000000000000` and I'm getting `Error: Unable to retrieve header and parent from supplied hash`

Reply:
It should never return genesis header, because in test there's a "init-bridge-hub-westend-local" call before this script is called, which means that at least one block is mined when this script starts. Are you running a test, or calling the script manually?

Reply:
I was running `./bridges/zombienet/run-tests.sh --test 2`

Reply:
Can you, please check if any blocks were produced for BH then? Maybe that's the reason. It could happen e.g. on Westend if you'll use `polkadot` binary, compiled without `--features fast-runtime`. I could also add explicit check for the `0x0000000000000000000000000000000000000000000000000000000000000000`, but it makes no sense - test will fail anyway